src/distributions/exponential.rs

```diff
@@ -30,10 +30,10 @@ use distributions::{ziggurat, ziggurat_tables, Distribution};
 ///
 /// # Example
 /// ```rust
-/// use rand::{NewRng, SmallRng, Rng};
+/// use rand::{thread_rng, Rng};
 /// use rand::distributions::Exp1;
 ///
-/// let val: f64 = SmallRng::new().sample(Exp1);
```

Comment:
Using `SmallRng` in these examples seems like something nice to preserve. It does not matter much, but I expect this code to be mostly used in combination with the simpler non-cryptographic RNGs. So it would fit more with the way users use it.

Reply:
Okay, fair point.